src/components/Navigation/Navigation.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import './Navigation.css';
+
+const Navigation = () => (
+  <nav className="navigation">
+    <a className="navigation__link" href="#about">About</a>
+    <a className="navigation__link" href="#services">Services</a>
+    <a className="navigation__link" href="#contact">Contact</a>
```

Comment:
```suggestion
      {links.map(link => (
        <a
          className="navigation__link"
          key={link.href}
          href={link.href}
        >
          {link.label}
        </a>
      ))}
```
Also in this case you can create some array, and to use map to avoid code duplication))